README.md

```diff
@@ -1,74 +1,159 @@
+All the remediation events of FAR (as well as other Medik8s operators) has a message that begins with *[remediation]*. Therefore, to easily filter these events run `oc get events -A | awk '/\[remediation\]/ || NR==1'` to get any remediation event or `oc get events -A | awk '/\[remediation\]/ && /worker-1/ || NR==1'` for getting any remediation event for node and CR of name *worker-1*.
 
 ## Installation
 
-There are two ways to install the operator:
+There are three ways to install the operator:
 
 * Deploy the latest version, which was built from the `main` branch, to a running Kubernetes/OpenShift cluster.
-<!-- TODO: - Deploy the last release version from OperatorHub to a running Kubernetes cluster. -->
-* Build and deploy from sources to a running or to be created Kubernetes/OpenShift cluster.
+* Deploy the latest release version from the Kubernetes community, [OperatorHub.io](https://operatorhub.io/operator/fence-agents-remediation), to a running Kubernetes cluster.
+* Build and deploy from sources to a running Kubernetes/OpenShift cluster.
 
 ### Deploy the latest version
 
 After every PR is merged to the `main` branch, then the images are built and pushed to [`quay.io`](quay.io/medik8s/fence-agents-remediation-operator-bundle) (due to the [*post-submit* job](https://github.com/medik8s/fence-agents-remediation/blob/main/.github/workflows/post-submit.yaml) ).
-For deployment of FAR using these images you need:
+For deployment of FAR using these images, you need:
 
 * Install `operator-sdk` binary from their [official website](https://sdk.operatorframework.io/docs/installation/#install-from-github-release).
 
-* A running OpenShift cluster, or a Kubernetes cluster with Operator Lifecycle Manager ([OLM](https://olm.operatorframework.io/docs/)) installed (to install it run `operator-sdk olm install`).
+* A running Kubernetes cluster, or an OpenShift (OCP) cluster with OLM installed. To install it on Kubernetes cluster run `operator-sdk olm install`.
+
+* A valid `$KUBECONFIG` is configured to access your cluster.
 
-* A valid `$KUBECONFIG` configured to access your cluster.
-<!-- TODO: ATM it can't be installed on the default namespace -->
-Then, run `operator-sdk run bundle quay.io/medik8s/fence-agents-remediation-operator-bundle:latest` to deploy the FAR's latest version on the current namespace.
+* Run `operator-sdk run bundle quay.io/medik8s/fence-agents-remediation-operator-bundle:latest` to deploy the FAR's latest version on the current namespace.
+  Another way to achieve that is running `BUNDLE_RUN_NAMESPACE=ns make bundle-run` to install FAR on *ns* namespace.
+
+> *Note*: Installing FAR on a new namesapce (e.g., ns) requires setting some labels on the namespace prior to installing FAR:
+kubectl label --overwrite ns olm security.openshift.io/scc.podSecurityLabelSync=false
+kubectl label --overwrite ns olm pod-security.kubernetes.io/enforce=privileged
```

Comment:
Please use code section

```bash
# your code here
```

Reply:
I can, but then it won't be under the same Note block.